test-clients/src/main/java/com/hedera/services/bdd/suites/suiterunner/TypedSuiteRunnerTest.java

```diff
@@ -0,0 +1,54 @@
+package com.hedera.services.bdd.suites.suiterunner;
+
+/*-
+ * ‌
+ * Hedera Services Test Clients
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.bdd.suites.HapiApiSuite;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+
+class TypedSuiteRunnerTest {
```

Comment:
Do we really need a test for the test runner? If the point is coverage - `test-clients` are **not** included in the codecov report.

Reply:
I never write tests only to satisfy the CI/CD requirements. 
The implementation is rather complex and I think that it would be nice for everyone to be confident, that the library does its job right, and that any further changes or refactorings do not break the implementation and the expected behavior.